### PR TITLE
[v1.2.5] Add and Delete Victim to opened Investigation Case

### LINKED ISSUE #5: Add Victims to Investigation Case
As an investigator, I can add a list of victims tied to an investigation case.

### LINKED ISSUE #16: Delete Victims from Investigation Case
As an investigator, I can delete victims tied to an investigation case.

### LINKED ISSUE #123: [v1.2.5] Add and Delete victim

### PR DESCRIPTION
**Pull request by constancensq**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -33,6 +34,21 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
...
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static Name parseName(String name) throws ParseException {
+        requireNonNull(name);
+        String trimmedTitle = name.trim();
+        if (!Name.isValidName(trimmedTitle)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
+        }
+        return new Name(name);
```
Comment (CodyChew) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Should be trimmedTitle here

**Comment by Vielheim** [comment]
Sorry I pressed Merge Commit by accident ><

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (Vielheim) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Updated

**Review by Vielheim — APPROVED** [review]
LGTM

**Pull request merged**